### PR TITLE
Cpp guards && recursive include cleanup

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: channels/remdesk/client/remdesk_main.c [hunk 1]
```diff
@@ -25,8 +25,10 @@
 #include <winpr/assert.h>
 #include <winpr/print.h>
 
+#include <freerdp/freerdp.h>
 #include <freerdp/assistance.h>
 
+#include <freerdp/freerdp.h>
```
Comment (hardening) [inline channels/remdesk/client/remdesk_main.c:31]: duplicate include

File: channels/printer/client/printer_main.c [hunk 2]
```diff
@@ -36,8 +36,10 @@
 #include <winpr/interlocked.h>
 #include <winpr/path.h>
 
+#include <freerdp/freerdp.h>
 #include <freerdp/channels/rdpdr.h>
 #include <freerdp/crypto/crypto.h>
+#include <freerdp/freerdp.h>
```
Comment (hardening) [inline channels/printer/client/printer_main.c:42]: duplicate include

File: channels/remdesk/client/remdesk_main.c (hunk 1 above)
Comment (akallabeth) [reply channels/remdesk/client/remdesk_main.c:31]: good catch

File: channels/printer/client/printer_main.c (hunk 2 above)
Comment (akallabeth) [reply channels/printer/client/printer_main.c:42]: good catch

**Review by hardening — APPROVED** [review]

**Pull request merged**